public/editor.html

```diff
@@ -204,6 +204,80 @@ <h1 class="mb-4 mt-2">
                     </div>
                   </div>
                 </div>
+                <br />
+                <div class="p-3 bg-warning bg-gradient">
+                  <div class="row">
+                    <div class="col-10"><h4>Aging</h4></div>
+                    <div class="col-2 text-right">
+                      <input
+                        class="form-check-input random-group randomize-smile-lines"
+                        type="checkbox"
+                        value=""
+                        checked
+                        id="randomize-aging"
+                      />
+                    </div>
+                  </div>
+                  <div class="row">
+                    <div class="col-10">
+                      <div class="form-group">
+                        <select class="form-control" id="aging-enabled">
+                        </select>
+                      </div>
+                    </div>
+                    <div class="col-2 text-right">
+                      &nbsp;
+                    </div>
+                  </div>
+                  <div class="row">
+                    <div class="col-5">
+                      <label for="aging-age">Age</label>
+                    </div>
+                    <div class="col-5 form-group">
+                      <input
+                        type="number"
```

Comment:
Change this to `type="range"` and you get a nifty slider. Should display the age as text next to it too.